src/clientdlg.cpp

```diff
@@ -84,11 +84,13 @@ CClientDlg::CClientDlg ( CClient*         pNCliP,
     lbrInputLevelL->setAccessibleName ( strInpLevHAccText );
     lbrInputLevelL->setAccessibleDescription ( strInpLevHAccDescr );
     lbrInputLevelL->setToolTip ( strInpLevHTT );
+    lbrInputLevelL->installEventFilter ( this ); // install event filter for tooltips
```

Comment:
The comment is basically a duplicate of the method. Not sure how much it helps.

Reply:
Not really: the supplied event filter gets called by Qt for every event - the comment is to clarify that the event filter provided on `this` only handles `toolTip` events (rather than, e.g. click events).